frontend/assets/javascripts/src/modules/metrics/triggers.js

```diff
@@ -13,7 +13,7 @@ define([
 
     function init() {
         var trackingElems = document.querySelectorAll('[' + METRIC_ATTRS.trigger + ']');
-        if (window.ga && trackingElems) {
+        if (trackingElems) {
```

Comment:
I'm not sure this is the right fix for this problem? `configureListeners(trackingElems);` presumably does want to wait until *both* `window.ga` and `trackingElems` are ready?

It ultimately calls the stuff in https://github.com/guardian/membership-frontend/blob/1ff87757b/frontend/assets/javascripts/src/modules/metrics/logger.js - which needs `ga` to be present by that point, otherwise it will be as ineffective as the current code?

I'd guess what you really need is a callback that gets called when `ga` loads?